lib/action_kit/client.rb

```diff
@@ -0,0 +1,17 @@
+module ActionKit
```

Comment:
Eric, I was mistaken - unfortunately the wrapper doesn't offer exactly what I need, so for now I'll keep my few lines which make the necessary API query.